spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestFilteredScan.java

```diff
@@ -600,10 +602,10 @@ private List<Record> expected(int... ordinals) {
     return expected;
   }
 
-  private void pushFilters(ScanBuilder scan, Filter... filters) {
-    Assertions.assertThat(scan).isInstanceOf(SupportsPushDownFilters.class);
-    SupportsPushDownFilters filterable = (SupportsPushDownFilters) scan;
-    filterable.pushFilters(filters);
+  private void pushFilters(ScanBuilder scan, Predicate... predicates) {
```

Comment:
Why add `.toV2` everywhere instead of just calling it here? Seems like this would be an easier place to update.

Reply:
Ha ha ha, it's true to be clean and easy with this. Let met change it.

Reply:
Updated